CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [Version 0.2.5](https://github.com/dataiku/dss-plugin-tableau-hyper/releases/tag/v0.2.5) - Feature release - 2024-06-04
+
+- Fix: Invalid page number when querying Tableau server
```

Comment:
Give more details

Reply:
Done in 75aab9dab2abca0220e1149bc82d53d9733d047f